source/SIL.AppBuilder.Portal.Frontend/src/ui/components/inputs/locale-input/-utils/localize.ts

```diff
@@ -27,14 +27,14 @@ export function localizeTagData(data: ILanguageInfo[], t): ILanguageInfo[] {
       //  - sldr: boolean;
       //  - tag: string;
       //  - tags?: string[];
+      //  - variants?: string[];
       nameInLocale: tryLocalize('languages', info.tag, info.name),
 
       // These are the only fields that need localization
       region: info.region && tryLocalize('territories', info.region, info.region),
       regions:
         info.regions &&
         info.regions
-          .split(' ')
           .map((region) => tryLocalize('territories', region, region))
           .join(', '),
```

Comment:
the type definition for ILanguageInfo previously defined regions as a string and now defines regions an an array of string
```
-  regions?: string;
+ regions?: string[];
```

```
.join(', ')
``` 
turns an array of string regions into a comma delimited string of regions which is not how the type is defined. which causes some issues in `helpers.tsx` since an array of string is expected based on the type but at runtime the property is made a string

Reply:
if you remove the `.join` a new error is thrown in `find-additional-matches.tsx`

I _think_ you can replace this code 

```
    let value = Array.isArray(rawValue) ? rawValue.join(', ') : rawValue;
    let num = Array.isArray(rawValue) ? rawValue.length : 1;

    if (rawValue && key === 'regions') {
      num = rawValue.split(' ').length;
    }
```

with this code
```
    const value = Array.isArray(rawValue) ? rawValue.join(', ') : rawValue;
    const num = Array.isArray(rawValue) ? rawValue.length : 1;
```

since regions is now an array we no longer need special handling to compute the count.  I am not sure if this will cause regressions anywhere else but this does seem to fix the problem for me

Reply:
Yes, we figured out that we needed to remove the `.join`.

Yes, we also remove the special case for regions in `find-additional-matches.tsx.